README.md

```diff
@@ -1,30 +1,47 @@
+[![JetBrains Research](https://jb.gg/badges/research.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
+
 # Codetracker
 
 **Codetracker** is an IntelliJ-based IDE plugin for tracking code changes while solving programming problems.
+The plugin is built by the [ML4SE lab @ JetBrains Research](https://research.jetbrains.org/groups/ml_methods) 
+to facilitate collection of data for research projects.
 
 ### Code changes tracking
 
-This plugin writes changes in all open documents in a .csv files and sends them after all to the [codetracker server](https://github.com/nbirillo/coding-assistant-server). 
-All data is sent **anonymously**, the only information is file names and the source code. This data can be helpful for 
-analyzing students programming patterns or collecting diverse solutions of given problems. For example, 
-see [automated hint generation system](https://github.com/JetBrains-Research/codetracker-org.jetbrains.research.ml.codetracker.data), 
+Codetracker tracks changes made in code during problem-solving.
```

Comment:
Maybe is it better to use `task-solving`? I am not sure the tasks and problem we can use interchangeably in this case